haskell/src/Ligo/BaseDAO/TZIP16Metadata.hs

```diff
@@ -205,6 +207,20 @@ getTotalSupplyView MetadataSettings{ msConfig = MetadataConfig{} } = View
       ]
   }
 
+tokenAddressView :: DaoView
+tokenAddressView MetadataSettings{} = View
+  { vName = "token_address"
+  , vDescription = Just
+      "Returns the address of the associated FA2 contract."
+  , vPure = Just True
+  , vImplementations =
+      [ VIMichelsonStorageView $
+          mkMichelsonStorageView @Storage Nothing [] $
+            unsafeCompileViewCode $ WithoutParam $ do
+              drop; selfAddress
```

Comment:
Ah, at the time this was removed `SELF` was indeed the address to the only meaningful token for baseDAO.

However since #170 we have now an external governance token, so I think we should change this view to return that (see the `storage.governance_token.address` field).

Perhaps it makes sense to rename the view as well? To `governance_token_address`?

Reply:
I see, I didn't check this properly. I have updated this now. Let me know if you want me to rename the commit as well.

Reply:
Maybe the view can fetch the governance token id as well? @pasqu4le ?

Reply:
I'm not quite familiar with the new governance token. But it does seem to make more sense to include them both if I'm not mistaken.

Reply:
Indeed, I agree that it would probably be more useful, let's return the whole governance token pair, both address and id :ok_hand:

Reply:
I've pushed an update to this now.